lldb/test/Shell/Settings/TestFrameFormatFunctionArguments.test

```diff
@@ -0,0 +1,60 @@
+# Test the ${function.arguments} frame-format variable.
```

Comment:
Probably better to factor this according to the language dimension, since the objc parts will most likely need to be disabled on non-macosx.

Reply:
done